### DISCUSSION TITLE
Corresponding brain plane of Nifti read [X, Y, Z] and its orientation

### DISCUSSION THREAD

**Discussion by zeno3175** [bjw34032/oro.nifti#10] [Q&A]
What are the corresponding brain planes: sagittal, coronal, axial plan regarding to X, Y, Z dimension as the following lines? Is the orientation following RAS or LPS rule?
dat<- readNIfTI("img.nii")
img<- dat@.Data
X<- dim(img)[1]
Y<- dim(img)[2]
Z<- dim(img)[3]

Thank you!

**Comment by bjw34032** [comment]
Hi.  Not sure I understand your question.  The common orientation would be for x = left-right, y = anterior-posterior and z = superior-inferior... but this depends on how the data were acquired.  The easiest thing to do would be to plot the data using

```oro.nifti::orthographic(dat)```

...and see what it produces.  There is also no need to extract the multidimensional array from the S4 object, methods will detect the `nifti` class and if they don't then they'll automatically treat it as an `array`.

Cheers, Brandon

**Reply by zeno3175** [reply]
Hi Brandon,  thanks first. I was trying to open the coordinates from the mask I generated from Nifiti label in "fsleyes". They have xyz for "Coordinates: Scanner anatomical" and "voxel location", I tried in both areas but didn't match with the label, so I was wondering if there is some transformation I need to do with the coordinates before I open it in fsleyes or other viewer?

Best, Qi

**Comment by bjw34032** [comment]
I have never used FSLeyes, sorry.  I use ITKSNAP for most of my visualization needs.  

Without knowing how the data were acquired and FSLeyes works it is difficult to know how to proceed.  If you are not satisfied with the orientation of the NIfTI file, then you could swap the dimensions using FSL, `aperm` in R or some functionality from `numpy` in Python I guess.  

Cheers, Brandon

**Reply by zeno3175** [reply]
Thanks, I will do that
